### PR TITLE
DM-36221: Add processing section 

### PR DESCRIPTION
**Pull request by hsinfang**

### CODE REVIEW

File: processing.tex [hunk 1]
```diff
@@ -1,5 +1,127 @@
...
+
+\end{itemize} %step1
+
+\item step2
+\begin{itemize}
+
+ \item
+ We expected that step2 would need a total of 3 workflows ($\approx$~6600 visits each), based on quantum graph generation tests, but execution butler creation became the bottleneck instead, and we instead required 14 smaller workflows ($\approx$~1500 visits each).
+ For example, for the first of these smaller step2 workflows, quantum graph generation took 23 min, but execution butler creation took 7 hours, while job compute time was 2.5 hours (wall clock).
+ Before step3 processing started, this issue was resolved in DM-33345, and execution butler generation time was dramatically improved, e.g., from 7 hours down to 20 minutes in this step2 example, or from 1 hour down to 6.5 minutes for a 1-tract step3 workflow.
```
Comment (timj) [inline processing.tex]: I think you can say `\jira{DM-33345}` to generate a link to the ticket.

Comment (timj) [reply processing.tex]: It will be even faster next time with graph-backed butler.

File: processing.tex [hunk 2]
```diff
@@ -1,5 +1,127 @@
...
+
+\end{itemize} %step2
+
+\item step3
+\begin{itemize}
+
+  \item
+  Large numbers of simultaneous assembleCoadd (= image coaddition) jobs led to ``too many request'' errors reading from the object store (PREOPS-1034) and revealed that data caching was being done inefficiently.
```
Comment (timj) [inline processing.tex]: Do we want to explain that this was because we coadd in chunks and loop over files for each chunk?

File: processing.tex [hunk 3]
```diff
@@ -1,5 +1,127 @@
...
+
+  \item
+  Long-running forcedPhotCoadd jobs were failing/re-attempting at a much higher rate than before, caused by a change in the PanDA pilot version, which reset the job heartbeat timeout limit back to its default of 2 hours, whereas it had been set to 20 hours previously.
+  This issue also uncovered the need for more frequent heartbeat message logging for long-running forcedPhotCoadd (DM-33854) and faro tasks (DM-33820).
+  The timeout limit was set again to 20 hours to allow step3 production to continue efficiently.
+
+  \item
+  About once every two workflows or so, a deblend job or two failed due to out-of-memory error (DM-33690), because of very bright/large objects on the coadd image.
+  These failures were fixed in rescue workflows, but required long run times ($>$10 hrs) and large memory ($\approx$~40 GB) using the \texttt{extra-highmem} queue, though ultimately with deblending results that were likely unreliable.
```
Comment (timj) [inline processing.tex:81]: We also had problems with pre-emption didn't we and had to switch to a non-pre-emptible queue.

File: processing.tex [hunk 4]
```diff
@@ -1,5 +1,127 @@
...
+50 to 80 to help stabilize the number of running jobs back to nearly 4000.
+
+
+\end{itemize} %step4
+
+\item step5
+\begin{itemize}
+
+  \item
+  We encountered a very slow start to jobs running on the PanDA system due to the large number of jobs per task ($>$100,000) in the first step5 workflow.
```
Comment (timj) [inline processing.tex:119]: At some point we needed to ask for an iDDS server with more resources and they also made it more efficient. I think we have been running the largest DAGs iDDS had had to handle.

**Review by timj — COMMENTED** [review]
Do I recall that we also had a problem with very slow merge jobs caused by having to check the existence of files in the object store multiple times? https://jira.lsstcorp.org/browse/DM-31785

File: processing.tex (hunk 4 above)
Comment (womullan) [reply processing.tex:119]: added a line

File: processing.tex (hunk 3 above)
Comment (hlin730) [reply processing.tex:81]: Yes, this was needed for last obstinate deblend quantum, and this is mentioned in the next line in the text.

File: processing.tex (hunk 2 above)
Comment (hsinfang) [reply processing.tex]: More details have been added in this paragraph. I think we can close this.

**Pull request merged**